src/mapping/config/MappingConfiguration.cpp

```diff
@@ -62,7 +64,10 @@ void addAttributes(TagStorage &storage, const std::vector<variant_t> &attributes
 // Enum required for the RBF instantiations
 enum struct RBFBackend {
   Eigen,
-  EigenGreedy,  //TODO: Das ist unschön.
+  FGreedyCut, // TODO: Just one "Greedy" type here? Differentiate via GreedyParameter?
```

Comment:
We can have multiple ones in the implementation (especially in an early development stage). It's a different story what we expose to the user configuration later on. I am only a bit afraid of code duplication across all the instantiations, but let's worry later. Maybe it turns out that we anyway only want one algorithm later on, because it delivers favorable properties.